shell/platform/linux/fl_mouse_cursor_plugin.cc

```diff
@@ -14,6 +14,12 @@ static constexpr char kBadArgumentsError[] = "Bad Arguments";
 static constexpr char kActivateSystemCursorMethod[] = "activateSystemCursor";
 static constexpr char kKindKey[] = "kind";
 
+static GHashTable* systemCursorTable = nullptr;
+
+bool define_system_cursor(const gchar *key, const gchar *value) {
```

Comment:
Needs a coment above the function. Function should be static. Private functions should be after the `G_DEFINE_TYPE`.